src/studio/kdb/Config.java

```diff
@@ -212,24 +214,42 @@ public void setNotesHash(String notesHash) {
         save();
     }
 
-    public Font getFont() {
-        String name = p.getProperty("font.name", "Monospaced");
-        int  size = Integer.parseInt(p.getProperty("font.size","14"));
+    private Font getFont(String prop) {
+        String name = p.getProperty(prop + ".name", "Monospaced");
+        int size = 14;
+        try {
+            size = Integer.parseInt(p.getProperty(prop + ".size", ""));
+        } catch (NumberFormatException e) {
+            // do nothing
+        }
 
-        Font f = new Font(name, Font.PLAIN, size);
-        setFont(f);
+        return new Font(name, Font.PLAIN, size);
+    }
 
-        return f;
+    private void setFont(String prop, Font f) {
+        p.setProperty(prop + ".name", f.getFontName());
+        p.setProperty(prop + ".size", Integer.toString(f.getSize()));
+        save();
     }
 
-    public String getEncoding() {
-        return p.getProperty("encoding", "UTF-8");
+    public Font getFont() {
+        return getFont("font");
     }
 
     public void setFont(Font f) {
-        p.setProperty("font.name", f.getFamily());
-        p.setProperty("font.size", "" + f.getSize());
-        save();
+        setFont("font", f);
+    }
+
+    public Font getTableFont() {
+        return getFont("tablefont");
+    }
+
+    public void setTableFont(Font f) {
+        setFont("tablefont", f);
+    }
```

Comment:
I wanted to reworked all properties to not have get<PropertyName>/set<PropertyName>. Instead I think to add get<Type>(String key)/set<Type>(String key, <value>). In the caller code, it would be `config.getFont(Config.TABLE_FONT)` and `config.setFont(Config.TABLE_FONT, newFont)`

Reply:
added enum